### PR TITLE
Adding CSS Environment Variables spec

### PR DESCRIPTION
**Pull request by nhoizey**

### CODE REVIEW

File: macros/SpecData.json [hunk 1]
```diff
@@ -194,6 +194,11 @@
     "url": "https://drafts.csswg.org/css-display/",
     "status": "WD"
   },
+  "CSS3 Environment Variables": {
+    "name": "CSS Environment Variables Module Level&nbsp;1",
+    "url": "https://drafts.csswg.org/css-env-1/",
+    "status": "WD"
```
Comment (JeremiePat) [inline macros/SpecData.json]: As this proposal is a draft, the value for `status` should be `Draft`

Comment (nhoizey) [reply macros/SpecData.json]: This is now fixed, thanks!

**Comment by JeremiePat** [comment]
LGTM
just to wait final review (I guess by @escattone, @jwhitlock, or @Elchi3)

**Comment by nhoizey** [comment]
Hi @escattone, @jwhitlock, do you see any issue with my PR? I can improve it, if you think it could be better. Thanks.

File: macros/SpecData.json [hunk 2]
```diff
@@ -194,6 +194,11 @@
     "url": "https://drafts.csswg.org/css-display/",
     "status": "WD"
   },
+  "CSS3 Environment Variables": {
+    "name": "CSS Environment Variables Module Level&nbsp;1",
+    "url": "https://drafts.csswg.org/css-env-1/",
+    "status": "Draft"
```
Comment (Elchi3) [inline macros/SpecData.json]: Should this be "ED" ?

File: macros/SpecData.json (hunk 1 above)
Comment (Elchi3) [reply macros/SpecData.json]: I think this should actually be "ED" for Editor's Draft instead.

Comment (nhoizey) [reply macros/SpecData.json]: @Elchi3 this is an unofficial draft, so is "ED" the same?

Comment (Elchi3) [reply macros/SpecData.json]: What does "unofficial draft" mean? It is presented as an ED and located under (I guess) an "official" domain https://drafts.csswg.org/

Comment (nhoizey) [reply macros/SpecData.json]: Ok, the red label on the left now says "Editor's draft", indeed! ;-)

**Comment by nhoizey** [comment]
@Elchi3 this is fixed

**Comment by nhoizey** [comment]
@Elchi3 awesome, thanks!

**Pull request merged**